web/pdf_page_view.js

```diff
@@ -187,6 +195,16 @@ class PDFPageView {
     }
   }
 
+  async _buildXfaTextContentItems(textDivs) {
+    const text = await this.pdfPage.getTextContent();
+    const items = [];
+    for (const item of text.items) {
+      items.push(item.str);
+    }
+    this.textHighlighter.setTextMapping(textDivs, items);
+    this.textHighlighter.enable();
```

Comment:
I'm not finding an corresponding `disable` call, for the xfaLayer-case, anywhere!?

Please note that the xfaLayer, similar to the textLayer, will now be removed from the DOM when the page becomes inactive (to reduce memory usage and improve performance).
Hence it seems to me that you'd want to, in the `cancelRendering`-method, ensure that you also *disable* the textHighlighter for the xfaLayer-case to properly clean-up here.